website/src/pages/license/index.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 3
+title: License
+description: Podman Desktop license
+tags: [license]
```

Comment:
should we have `licence` as well as it seems people are searching for this variation
so in tags and keywords ?

Reply:
I would say no because:

1. "licence" is french, and we currently have no plans regarding translation.

2. The search term is not widely used. In Google search, during the last 3 months, we had:
   - "podman desktop license": 24 hits / 211 search / ranked between 100 and 200.
   - "podman license": 13 hits / 605 search for / ranked between 200 and 300.
   - "podman license": 1 hit / 49 search / ranked 700 and 800.